### PR TITLE
Fix colors, details in view

### PR DESCRIPTION
**Pull request by paul-nameless**

### CODE REVIEW

File: tg/views/__init__.py [hunk 1]
```diff
@@ -129,32 +129,43 @@ def resize(self, p: float = 0.25) -> None:
         self.win.resize(self.h, self.w)
 
     def _msg_color(self, is_selected: bool = False) -> int:
-        return get_color(white, -1) | (reverse if is_selected else 0)
+        color = get_color(white, -1)
+        if is_selected:
```
Comment (lightme16) [inline tg/views/__init__.py:133]: why did you remove inline selection? `| 0` has no any effects. its logical OR with 0, so left operat should always be preserved.
``` 
> get_color(white, -1)
512
> get_color(white, -1) | 0
512
```

Comment (paul-nameless) [reply tg/views/__init__.py:133]: I think it's more readable and there are less or operations

Comment (lightme16) [reply tg/views/__init__.py:133]: okey, feel free to merge then
